gossipsub.go

```diff
@@ -27,7 +27,8 @@ var (
 	GossipSubHistoryGossip = 3
 
 	// heartbeat interval
-	GossipSubHeartbeatInterval = 1 * time.Second
+	GossipSubHeartbeatInitialDelay = 100 * time.Millisecond
```

Comment:
Just a quick question, in the tests I saw `100 * time.Millisecond` many times. Is it a measured safe delay for initial connections between peers and all the other needed functionalities in host and Swarm to set up? Thank you

Reply:
it's probably over-generous and not really measured, but it's a ballparked initialization time for stuff.
You can probably get away with smaller delays most of the time, but the gossipsub tests make a lot of peers.

Reply:
It makes a lot of sense. Thank you for the explanation.